### DISCUSSION TITLE
8주차 미팅

### DISCUSSION THREAD

**Discussion by nyj001012** [GunGonGamLee/ft_transcendence#213] [General]
8주차 목표 작성

**Comment by nyj001012** [comment]
# 프론트
## TODO
### 1. 백엔드 API 이식
- header 완료
- histories 진행중

### 2. 기획
#### 2.1. 게임 UI
- 게임 중 UI
- 게임 결과 UI
- 대진표 매칭 UI

#### 2.2. 아바타 변경 UI
- 마이페이지에서 아바타 변경 가능해야 함. 사용자가 임의로 업로드도 가능해야 함.

### 3. 리팩토링
- `useState` 사용  https://github.com/GunGonGamLee/ft_transcendence/pull/163
- `useEffect` 사용 https://github.com/GunGonGamLee/ft_transcendence/pull/201
- dev, prod 모드 분리 https://github.com/GunGonGamLee/ft_transcendence/pull/209

## 8주차 목표
### 프론트엔드
- 전체: 기획 완료
- yena: histories 항목 구현

**Comment by bluedog129** [comment]
# Todo

## 구현
### 친구 모달 구현
- [ ] UI
- [ ] 이벤트
- [ ] API 적용

### 대진표매칭 구현
- [ ] UI
- [ ] 이벤트
- [ ] API 적용

## 기획
- 게임 중 UI
- 게임 결과 UI
- 대진표 매칭 UI
